scripts/mlp/utils/plot.py

```diff
@@ -83,7 +83,7 @@ def plotEffectorRef(dict_refs,dt):
 def plotEffectorTraj(timeline,p_intervals,ref_dicts,traj_dicts):
     labels=["x (m)" , "y (m)" ,"z (m)", "dx (m/s)" , "dy (m/s)" ,"dz (m/s)","ddx (m/s^2)" , "ddy (m/s^2)" ,"ddz (m/s^2)"]
     colors = ['r','g','b']
-    for eeName in traj_dicts[0].keys():
+    for eeName in list(traj_dicts[0].keys()):
```

Comment:
Same here. The `.iteritems()` were about using iterators instead of lists, but in python3 you can directly get the `.items()` which will provide an iterator. So for the keys, you can also use an iterator in a for loop :)